src/EventInserterPQ.cpp

```diff
@@ -0,0 +1,113 @@
+#include <iostream>
+#include <string>
+#include <vector>
+#include <map>
+#include <libpq-fe.h>
+
+//#include "Event.h"
+#include "EventInserterPQ.h"
+
+using std::string;
+
+EventInserterPQ::EventInserterPQ(std::string db_name_, std::string table_name_, std::map<std::string, std::vector<std::string> > val_name_){
+	db_name = db_name_;
+	table_name = table_name_;
+	val_name = val_name_;
+}
+
+EventInserterPQ::~EventInserterPQ(){
+	// delete pointers
+	PQfinish(conn);
+	PQclear(res);
+}
+
+void EventInserterPQ::CreateTable(){
+	//if db_name not exist, create it; else do nothing
+	conn = PQsetdbLogin(pghost, pgport, pgoptions, pgtty, db_name.c_str(), login, pwd);
+	if(PQstatus(conn) != CONNECTION_OK)
```

Comment:
I know this is also a question of style, but let's be consistent across the code base. use:

if (condition) {
} else if (...) {
} ... {
} else {
}

i.e. braces around even one-liners